tests/FSharpLint.Console.Tests/TestApp.fs

```diff
@@ -157,3 +190,23 @@ module Fall =
 
         Assert.AreEqual(2, exitCode)
         Assert.AreEqual(sourceCode, File.ReadAllText input.FileName)
+
+        Assert.AreEqual(sourceCode, File.ReadAllText input.FileName)
+
+    [<Test>]
+    member __.``Lint source with fix option no need for fix with check option``() =
+        let sourceCode = """
+module Fass =
+    let foo = System.Collections.Generic.Dictionary<string, string>() |> ignore
+    let goo = Guid() |> ignore
+    let ntoo = Int32() |> ignore
+module Fall =
+    let uoo = Uid() |> ignore
+    let version =  System.Version()
+    let xoo = Uint32() |> ignore
+        """
+        let ruleName = "RedundantNewKeyword"
+        use input = new TemporaryFile(sourceCode, "fs")
+        let (exitCode, errors) = main [| "fix"; ruleName; input.FileName; "--check" |]
+
+        Assert.AreEqual(0, exitCode)
```

Comment:
should we make the Enum public so that you can use it here? btw: (nit) put an EOL at EOF always

Reply:
hmmm yup it's possible, I thaught exitCodes are handled easier if they are integers.

Reply:
I didn't suggest not using integers, why do you say that?

Reply:
No it wasn't your suggestion, It was just a misunderstanding carried from my old works.